packages/coq-of-ocaml/coq-of-ocaml.2.3.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "dev@clarus.me"
+homepage: "https://github.com/clarus/coq-of-ocaml"
+dev-repo: "git+https://github.com/clarus/coq-of-ocaml.git"
+bug-reports: "https://github.com/clarus/coq-of-ocaml/issues"
+authors: ["Guillaume Claret"]
+license: "MIT"
+build: [
+  ["sh" "-c" "cd OCaml && ./configure.sh"] {coq:installed}
+  [make "-C" "OCaml" "-j%{jobs}%"] {coq:installed}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+install: [
+  [make "-C" "OCaml" "install"] {coq:installed}
+]
+depends: [
+  "dune" {>= "1.11"}
```

Comment:
```suggestion
  "dune" {>= "2.0"}
```
According to the dune-project file this now requires 2.0

Reply:
@clarus can you please upstream this change?

Reply:
Thanks! I did upstream the change in https://github.com/clarus/coq-of-ocaml/pull/152